scripts/hudson/narayana.sh

```diff
@@ -353,53 +353,37 @@ function build_narayana {
 }
 
 function clone_as {
-  echo "Cloning AS sources from https://github.com/jbosstm/jboss-as.git"
+  repo="https://github.com/wildfly/wildfly.git" # was https://github.com/jbosstm/jboss-as.git
+  AS_BRANCH="26.x"
+  echo "Cloning AS sources from $repo"
 
   cd ${WORKSPACE}
   if [ -d jboss-as ]; then
     echo "Updating existing checkout of WildFly"
     cd jboss-as
 
-    git remote | grep upstream
```

Comment:
Why didn't you just leave these lines here and the lines 385-386? Was it cleanup? If so my comment around 383 I think is relevant still in case there already is a jboss-as directory